nativelib/src/main/resources/scala-native/time_millis.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+#include <sys/time.h>
+
+long long scalanative_current_time_millis() {
+    long long current_time_millis;
+
+#define MILLIS_PER_SEC 1000LL
+#define MICROS_PER_MILLI 1000LL
+
+    struct timeval tv;
+    gettimeofday(&tv, NULL);
```

Comment:
To not distract the major intent of this PR, I opened Issue #1948 to capture the fact that the return value of gettimeofday() is never checked.  This is a bug from the code before this port, not this PR, so it should not hold up this PR.